src/main/app/access-code/AccessCodePostController.ts

```diff
@@ -2,10 +2,10 @@ import { Logger } from '@hmcts/nodejs-logging';
 import autobind from 'autobind-decorator';
```

Comment:
I'm thinking it might be worth Elis regression testing the standard AoS linking of the respondent as part of the ticket. I can't see it in the ACs at the moment